sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/AadResourceServerWebSecurityConfigurerAdapter.java

```diff
@@ -30,10 +36,17 @@ protected void configure(HttpSecurity http) throws Exception {
         // @formatter:off
         http.oauth2ResourceServer()
             .jwt()
-            .jwtAuthenticationConverter(
-                new AadJwtBearerTokenAuthenticationConverter(
-                    properties.getPrincipalClaimName(), properties.getClaimToAuthorityPrefixMap()));
+            .jwtAuthenticationConverter(jwtAuthenticationConverter());
         // @formatter:off
     }
 
+    private Converter<Jwt, AbstractAuthenticationToken> jwtAuthenticationConverter() {
```

Comment:
Can we define a bean of this object?

Reply:
It's not necessary, this adapter is built-in in our starter, users can re-define a new one to custom the adapter without this converter.